client/src/ui.rs

```diff
@@ -1115,3 +1102,12 @@ fn lobby_slot_checkbox(
         }
     }
 }
+
+fn disconnect_client(
```

Comment:
irgwie fühlt sich dieses system hier nicht richtig an

Reply:
I see what you mean. I think we can move this 
```rust
multiplayer_roles.set(MultiplayerRoles::NotInGame);
```
 line out of here when we implement the ability to leave a game.